reference/constraints/Traverse.rst

```diff
@@ -0,0 +1,94 @@
+Traverse
+========
+
+Objects do not validate nested objects by default unless explicitly using this constraint.
+If only certain nested objects should be validated by cascade, consider using the :doc:`references/constraints/Valid`
```

Comment:
I think `specific` is preferable over `certain` in this context.

Reply:
Done, thanks!